Fix mismatched hints for Pizza Pi exercise (#935)
exercism/php: exercises/concept/pizza-pi/.docs/hints.md

## 1. A Dough Ratio

- This task uses basic arithmetic with addition and multiplication.

## 2. A Splash of Sauce

- To ensure a whole number of cans, you'll need to round up. PHP provides [numeric functions][numeric-functions] that can help.

## 3. Some Cheese, Please

- You can cube a number using the [exponentiation operator][arithmetic-operations].

> - You’ll need to round the result down; PHP has a [numeric function][numeric-functions] for that.

Reviewer: ```suggestion
- To ensure a whole number of pizzas, you’ll need to round down.
- PHP provides [numeric functions][numeric-functions] that can help.
```

> - PHP also provides a [built-in function][pi-function] for pi (π) 

Reviewer: Usually the constant is prefered, as it avoids the function call overhead.

```suggestion
- PHP has a [built-in constant][pi-constant] for pi (π).
```
